### DISCUSSION TITLE
Frontend Initiatives Meeting: January 26, 2021

### DISCUSSION THREAD

**Discussion by amcgee** [dhis2/notes#199] [Meetings] [answered]
When: January 26, 2021 @ 15:00 CET
Where : Remote

Agenda
- General updates:
  - Welcome back @varl
- Updates and proposals:
  - [UI Docs - renovation](https://github.com/dhis2/notes/discussions/197) @KaiVandivier
  - Developer outreach and "single entry point" @deboragaleano @amcgee 
    - Upcoming academy:
      - Webinar Feb 8
      - Workshop March 17-18
      - Workshop May 4-7
      - Annual Conference June 21-25
  - [PR Preview](https://github.com/dhis2/notes/discussions/183) next steps @paschalidi 
  - How is the new JIRA structure / consolidation working?
  - (if time) Dynamic runtime modules @amcgee 
  - Others?  @dhis2/front-end
  
Will keep this meeting short and focused on a few topics, meeting again next week (February 3) for working group updates, so take a look at [last meeting's notes](https://github.com/dhis2/notes/blob/master/decisions/2021/01/14-frontend-initiatives.md) so you can share an update

**Comment by paschalidi** [comment]
@amcgee was curious if you can add the PR preview again so that we can discuss next steps?

**Comment by varl** [comment]
I'd like to know how the JIRA changes are working (CLI / LIBS / No-GH-issues).

**Comment by varl** [comment, marked as answer]
Meeting notes: https://github.com/dhis2/notes/pull/203
